{"uid":"9763d6565cddd00b","name":"TC 04_01_07 Verify Last name input field highlighting on label click","fullName":"tests.test_create_an_account.TestCreateAnAccount#test_04_01_04_lastname_highlighted_by_label","historyId":"4e8569cfeebd2604e72a526b596104bf","time":{"start":1730856986062,"stop":1730856986062,"duration":0},"description":" Verify that Last name input field highlighted when label is clicked","descriptionHtml":"<p>Verify that Last name input field highlighted when label is clicked</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.622\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x561c4418f0aa <unknown>\n#1 0x561c43ca61a0 <unknown>\n#2 0x561c43c8e840 <unknown>\n#3 0x561c43c8e52d <unknown>\n#4 0x561c43c8c5bc <unknown>\n#5 0x561c43c8cd6f <unknown>\n#6 0x561c43c9c1d7 <unknown>\n#7 0x561c43cb21e4 <unknown>\n#8 0x561c43cb751b <unknown>\n#9 0x561c43c8d47e <unknown>\n#10 0x561c43cb1fee <unknown>\n#11 0x561c43d35acb <unknown>\n#12 0x561c43d16b23 <unknown>\n#13 0x561c43ce5990 <unknown>\n#14 0x561c43ce696e <unknown>\n#15 0x561c4415b16b <unknown>\n#16 0x561c4415ef68 <unknown>\n#17 0x561c4414864c <unknown>\n#18 0x561c4415fae7 <unknown>\n#19 0x561c4412d4af <unknown>\n#20 0x561c4417d4f8 <unknown>\n#21 0x561c4417d6c0 <unknown>\n#22 0x561c4418df26 <unknown>\n#23 0x7f4e75294ac3 <unknown>","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"1132e83f2a7cc27d67e53d4a46b080eb\")>\n\n    @pytest.fixture(scope=\"function\")\n    def create_account_page(driver):\n        create_account_page = CreateAccountPage(driver, CREATE_ACCOUNT_PAGE_URL)\n>       create_account_page.open()\n\ntests/test_create_an_account.py:12: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/base_page.py:19: in open\n    self.driver.get(self.url)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd152675c10>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"timeout\",\"message\":\"timeout: Timed out receiving message from renderer: 2...\\\\n#21 0x561c4417d6c0 \\\\u003Cunknown>\\\\n#22 0x561c4418df26 \\\\u003Cunknown>\\\\n#23 0x7f4e75294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.622\nE         (Session info: chrome=130.0.6723.58)\nE       Stacktrace:\nE       #0 0x561c4418f0aa <unknown>\nE       #1 0x561c43ca61a0 <unknown>\nE       #2 0x561c43c8e840 <unknown>\nE       #3 0x561c43c8e52d <unknown>\nE       #4 0x561c43c8c5bc <unknown>\nE       #5 0x561c43c8cd6f <unknown>\nE       #6 0x561c43c9c1d7 <unknown>\nE       #7 0x561c43cb21e4 <unknown>\nE       #8 0x561c43cb751b <unknown>\nE       #9 0x561c43c8d47e <unknown>\nE       #10 0x561c43cb1fee <unknown>\nE       #11 0x561c43d35acb <unknown>\nE       #12 0x561c43d16b23 <unknown>\nE       #13 0x561c43ce5990 <unknown>\nE       #14 0x561c43ce696e <unknown>\nE       #15 0x561c4415b16b <unknown>\nE       #16 0x561c4415ef68 <unknown>\nE       #17 0x561c4414864c <unknown>\nE       #18 0x561c4415fae7 <unknown>\nE       #19 0x561c4412d4af <unknown>\nE       #20 0x561c4417d4f8 <unknown>\nE       #21 0x561c4417d6c0 <unknown>\nE       #22 0x561c4418df26 <unknown>\nE       #23 0x7f4e75294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: TimeoutException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1730856963016,"stop":1730856963081,"duration":65},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1730856963081,"stop":1730856963081,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1730856986063,"stop":1730856987324,"duration":1261},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"create_account_page","time":{"start":1730856987324,"stop":1730857287342,"duration":300018},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.622\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x561c4418f0aa <unknown>\n#1 0x561c43ca61a0 <unknown>\n#2 0x561c43c8e840 <unknown>\n#3 0x561c43c8e52d <unknown>\n#4 0x561c43c8c5bc <unknown>\n#5 0x561c43c8cd6f <unknown>\n#6 0x561c43c9c1d7 <unknown>\n#7 0x561c43cb21e4 <unknown>\n#8 0x561c43cb751b <unknown>\n#9 0x561c43c8d47e <unknown>\n#10 0x561c43cb1fee <unknown>\n#11 0x561c43d35acb <unknown>\n#12 0x561c43d16b23 <unknown>\n#13 0x561c43ce5990 <unknown>\n#14 0x561c43ce696e <unknown>\n#15 0x561c4415b16b <unknown>\n#16 0x561c4415ef68 <unknown>\n#17 0x561c4414864c <unknown>\n#18 0x561c4415fae7 <unknown>\n#19 0x561c4412d4af <unknown>\n#20 0x561c4417d4f8 <unknown>\n#21 0x561c4417d6c0 <unknown>\n#22 0x561c4418df26 <unknown>\n#23 0x7f4e75294ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/tests/test_create_an_account.py\", line 12, in create_account_page\n    create_account_page.open()\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 19, in open\n    self.driver.get(self.url)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Open a browser","time":{"start":1730856987324,"stop":1730857287342,"duration":300018},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.622\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x561c4418f0aa <unknown>\n#1 0x561c43ca61a0 <unknown>\n#2 0x561c43c8e840 <unknown>\n#3 0x561c43c8e52d <unknown>\n#4 0x561c43c8c5bc <unknown>\n#5 0x561c43c8cd6f <unknown>\n#6 0x561c43c9c1d7 <unknown>\n#7 0x561c43cb21e4 <unknown>\n#8 0x561c43cb751b <unknown>\n#9 0x561c43c8d47e <unknown>\n#10 0x561c43cb1fee <unknown>\n#11 0x561c43d35acb <unknown>\n#12 0x561c43d16b23 <unknown>\n#13 0x561c43ce5990 <unknown>\n#14 0x561c43ce696e <unknown>\n#15 0x561c4415b16b <unknown>\n#16 0x561c4415ef68 <unknown>\n#17 0x561c4414864c <unknown>\n#18 0x561c4415fae7 <unknown>\n#19 0x561c4412d4af <unknown>\n#20 0x561c4417d4f8 <unknown>\n#21 0x561c4417d6c0 <unknown>\n#22 0x561c4418df26 <unknown>\n#23 0x7f4e75294ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 19, in open\n    self.driver.get(self.url)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1730857287399,"stop":1730857287504,"duration":105},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1730858923681,"stop":1730858923681,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Create An Account"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_create_an_account"},{"name":"subSuite","value":"TestCreateAnAccount"},{"name":"host","value":"fv-az711-519"},{"name":"thread","value":"3043-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_create_an_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"d584cdba40da11d","reportUrl":"https://verafes.github.io/luma_project/485//#testresult/d584cdba40da11d","status":"passed","time":{"start":1730770617394,"stop":1730770618043,"duration":649}},{"uid":"6aa2802653d35b71","reportUrl":"https://verafes.github.io/luma_project/484//#testresult/6aa2802653d35b71","status":"passed","time":{"start":1730684573357,"stop":1730684574058,"duration":701}},{"uid":"930166639c044495","reportUrl":"https://verafes.github.io/luma_project/483//#testresult/930166639c044495","status":"passed","time":{"start":1730598255922,"stop":1730598256618,"duration":696}}]},"tags":[]},"source":"9763d6565cddd00b.json","parameterValues":[]}